### DISCUSSION TITLE
phobos 1.0.0

### DISCUSSION THREAD

**Discussion by Zane2b2t** [3arthqu4ke/3arthh4ck#214] [General]
i'd dm u if u didnt delete ur public acc but
![unknown-67](https://user-images.githubusercontent.com/88916121/188225586-db91be6e-7e7e-4840-aa56-fe0b5971a587.png)
this photo was taken TODAY and im also in the chat <PlzDontClaimNxme> gg

**Comment by Zane2b2t** [comment]
![Capture](https://user-images.githubusercontent.com/88916121/188225779-a7dc784a-2e1e-4713-99b8-0373e3e414d4.PNG)

**Reply by Zane2b2t** [reply]
lolwut2 is the one who sent it

**Comment by GurteX** [comment]
so?

**Reply by Zane2b2t** [reply]
> so?

phobos 1.0.0 not public and lolwut2 isnt a phobos

**Reply by GurteX** [reply]
you can actually find it thats not super hard

**Comment by 3arthqu4ke** [comment]
Oh no what am I going to do! Time to quit crystalpvp and play [crystalpvp 2](https://3arthqu4ke.github.io/3arthh4ck/)!

**Reply by Zane2b2t** [reply]
LMAO but no thats not what i meant i mean where phobos 1.0.0 me need :pray:
